src/ui/screens/onboarding/OnboardingPrivacySlide.qml

```diff
@@ -0,0 +1,78 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.5
+import QtQuick.Layouts 1.15
+
+import Mozilla.Shared 1.0
+import components 0.1
+import "qrc:/ui/screens/settings/privacy"
+
+ColumnLayout {
+    id: root
+
+    signal nextClicked()
+    signal backClicked()
+
+    spacing: 0
+
+    MZHeadline {
+        Layout.topMargin: MZTheme.theme.windowMargin * 1.5
+        Layout.leftMargin: MZTheme.theme.windowMargin * 2
+        Layout.rightMargin: MZTheme.theme.windowMargin * 2
+
+        Layout.fillWidth: true
+
+        text: MZI18n.OnboardingPrivacySlideHeader
+        horizontalAlignment: Text.AlignLeft
+    }
+
+    MZInterLabel {
+        Layout.topMargin: 8
+        Layout.leftMargin: MZTheme.theme.windowMargin * 2
+        Layout.rightMargin: MZTheme.theme.windowMargin * 2
+        Layout.fillWidth: true
+
+        text: MZI18n.OnboardingPrivacySlideBody
+        horizontalAlignment: Text.AlignLeft
+        color: MZTheme.theme.fontColor
+    }
+
+    PrivacyFeaturesList {
+        Layout.topMargin: 24
+        Layout.leftMargin: 32
+        Layout.rightMargin: 32
+        Layout.fillWidth: true
+    }
+
+    Item {
+        Layout.fillHeight: true
+        Layout.minimumHeight: 20
+    }
+
+    MZButton {
+        Layout.leftMargin: MZTheme.theme.windowMargin * 2
```

Comment:
```suggestion
        id: nextButton
        Layout.leftMargin: MZTheme.theme.windowMargin * 2
```

Reply:
I don't normally provide id's to components unless they need to be referenced, are you suggesting otherwise?

Reply:
From my past experiences I thought for better readability we should always add `id` but I just did some Googling and that definitely doesn't seem to be the standard so please disregard.